core/editor.js

```diff
@@ -1305,9 +1305,25 @@
 
 		/**
 		 * Moves the selection focus to the editing area space in the editor.
+		 *
+		 * Takes optional param `focusOptions` which holds property `preventScroll`. If set to true will prevent scrolling on focus.
+		 * Another property of `focusOptions` is `defer`. If set to true focus will be delayed by 100ms.
+		 *
+		 * Example:
+		 * ```javascript
+		 * editor.focus( { preventScroll: true } ); // Focuses but prevents scrolling.
+		 * editor.focus( { preventScroll: false } ); // Focuses without preventing scroll.
+		 * editor.focus( { defer: false } ); // Focuses after 100ms.
+		 * editor.focus(); // Focuses without preventing scroll.
+		 * ```
+		 *
+		 * @param {Object} [focusOptions] Focus options since **4.11.0**.
```

Comment:
We describe `focusOptions` in three different places, everywhere the same way.

It seems that we need… [DU-DU-DUUUUUUM](https://www.youtube.com/watch?v=UgwSSAzxcs8) dedicated type for it.